roborock/roborock_queue.py

```diff
@@ -12,7 +13,9 @@ def __init__(self, protocol: int, *args):
         self.protocol = protocol
 
     async def async_put(self, item: tuple[Any, RoborockException | None], timeout: float | int) -> None:
-        return await asyncio.wait_for(self.put(item), timeout=timeout)
+        async with async_timeout.timeout(timeout):
+            await self.put(item)
```

Comment:
We can put return back here if you would like, but I think it is not needed, since put does not return anything